apps/docs/components/blocks/Gallery.md

```diff
@@ -7,6 +7,10 @@ hideToc: true
 ---
 # Gallery
 
+::: warning This is an experimental block
+The API of this block might change based on user feedback.
```

Comment:
As you said, this is a block and all blocks may change. Also blocks does not expose an API, so I'm not sure if it's right to say that. 🤔 Maybe better way would be to say that this block is build on top of the experimental features?

Reply:
I've changed it after consult with Jacek & Michał:
2d1e20dd1662394d024bb001a167c75cb04ded14

Reply:
That's better. 👍🏼